### ISSUE TITLE
no `u32` in the root

### ISSUE THREAD

**Issue by ctaggart** [ctaggart/octh#14]
```
error[E0432]: unresolved import `self::super::super::u32`
    --> src\lib.rs:3697:17
     |
3697 |         pub use self::super::super::u32 as ios_base_fmtflags;
     |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `u32` in the root

error[E0432]: unresolved import `self::super::super::u32`
    --> src\lib.rs:3698:17
     |
3698 |         pub use self::super::super::u32 as ios_base_iostate;
     |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `u32` in the root

error[E0432]: unresolved import `self::super::super::u32`
    --> src\lib.rs:3699:17
     |
3699 |         pub use self::super::super::u32 as ios_base_openmode;
     |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `u32` in the root

error[E0432]: unresolved import `self::super::super::u32`
    --> src\lib.rs:3700:17
     |
3700 |         pub use self::super::super::u32 as ios_base_seekdir;
     |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `u32` in the root
```

**Comment by ctaggart** [comment]
fixed by #22

**Comment by ctaggart** [comment]
Still causing issues, but working on it in #15.
